internal/handlers/pg/pgdb/query.go

```diff
@@ -251,12 +251,13 @@ func prepareWhereClause(sqlFilters *types.Document) (string, []any, error) {
 			return "", nil, lazyerrors.Error(err)
 		}
 
-		if k != "" {
+		switch {
+		case k == "":
```

Comment:
How is that possible?

Reply:
What exactly? Empty keys?

Here's small test from mongodb:
```
test> db.values.insertOne({"":1})
{
  acknowledged: true,
  insertedId: ObjectId("6401bf27047d38a39d55655c")
}
test> db.values.find({"":1})
[ { _id: ObjectId("6401bf27047d38a39d55655c"), '': 1 } ]
test>
```

Reply:
So we have a test for that?

Reply:
https://github.com/FerretDB/FerretDB/blob/1f076a5d8b3fd006a6f8570c593fdbc4518b1cf0/integration/basic_test.go#L231

Reply:
Should that test check that pushdown was performed?

Reply:
We can do it.